### PR TITLE
Update models.py

### PR DESCRIPTION
**Pull request by sdgds**
1. change the 'dnn_train_model' and 'dnn_test_model' to realize the function that caculate loss and accuracy every epoch.
2. the accuracy involve 'ACC_train_top1, ACC_train_top5, ACC_val_top1, ACC_val_top5' as a value of dict when 'EPOCH' is the key of dict.

### CODE REVIEW

**Review by helloTC — CHANGES_REQUESTED** [review]
1 Delete matplotlib for this module didn't need it.
2 Why do you need to input three dataloaders as parameters of dnn_train_model: dataloaders_train, dataloaders_train_test, dataloaders_val_test? Moreover, please follow our previous discussion to set validation dataloader as an  optional parameter (if you actually need dataloaders_train_test, please discuss with me for how and whether to set it as optional parameters).
3 Make sure you have tested your code so we can directly use it without any bugs.
4 Please finish all annotations because you have modified the interface, especially I noted you didn't update annotation for the output of dnn_test_model.

**Comment by zhenzonglei** [comment]
What did happen for this pull request?

**Comment by helloTC** [comment]
> What did happen for this pull request?

Yiyuan is modifying the way to train a new model. In his modification, he wanted to combine function of dnn_test_model into the function of dnn_train_model to allow do validation by using validation data in each epoch. Moreover, due to my previous version only output top1 accuracy, in his update version he added top5 accuracy as one of the outputs. 
My previous comment on his pulling request required him make the way (e.g. training with validation) as an optional way because if users have no validation dataset, they can still train their deep learning model (actually a very large sample size of training data makes overfitting as a good thing in training a deep neural network). In addition, I required him clear some inconsistent code (e.g. code for plot).
